core/src/main/java/io/grpc/internal/ServerImpl.java

```diff
@@ -79,9 +79,10 @@
 public final class ServerImpl extends io.grpc.Server {
   private static final ServerStreamListener NOOP_LISTENER = new NoopListener();
 
-  /** Executor for application processing. */
+  /** Executor for application processing. Safe to read after {@link #start()}. */
```

Comment:
Based on our offline conversation, please add an issue to follow up on the documentation of these fields.